appserver/tests/payara-samples/samples/ejb-http-remoting/pom.xml

```diff
@@ -100,4 +100,49 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>3.1.0</version>
+                <executions>
+                    <execution>
+                        <id>setup-ejb-invoker</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>${payara.home}/bin/asadmin</executable>
```

Comment:
Ties the test specifically to Linux/Mac.

Asadmin needs to be a variable to allow the tests to run on Windows.
Example from Jakarta TCK runners:

```
<payara.asadmin>${payara.home}${file.separator}glassfish${file.separator}bin${file.separator}asadmin${script.extension}</payara.asadmin>

...

        <profile>
            <id>windows</id>
            <activation>
                <os>
                    <family>Windows</family>
                </os>
            </activation>
            <properties>
...
                <script.extension>.bat</script.extension>
...
            </properties>
        </profile>
```